tests/CMakeLists.txt

```diff
@@ -93,7 +93,7 @@ if(FTP_TEST_FILES)
     blend.t19z.core.f001.co.grib2
     aqm.t12z.max_8hr_o3.227.grib2 
     seaice.t00z.grb.grib2
-    cmc_geavg.t12z.pgrb2a.0p50.f000
+    #cmc_geavg.t12z.pgrb2a.0p50.f000
```

Comment:
Did you turn off a test file here?

Reply:
Yes, I will put it back. The CI was having some issues with downloading the test files and I assumed that it was the problem with one file, but I think it may have just been a temporary issue with the FTP site.